templates/storyboards/swift3.stencil

```diff
@@ -24,89 +20,74 @@ protocol StoryboardSceneType {
 }
 
 extension StoryboardSceneType {
-  static func storyboard() -> UIStoryboard {
-    return UIStoryboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
-  }
-
-  static func initialViewController() -> UIViewController {
-    guard let vc = storyboard().instantiateInitialViewController() else {
-      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
-    }
-    return vc
+  static func storyboard() -> {{prefix}}Storyboard {
+    return {{prefix}}Storyboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
   }
 }
 
 extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
But even for scenes we might want to have a reference to a scene without instantiating the VC just yet?

### Example enum usage for Segues

https://github.com/SwiftGen/SwiftGen/blob/master/SwiftGen.playground/Pages/Storyboards-Demo.xcplaygroundpage/Contents.swift#L12-L129

```swift
override func prepareForSegue(_ segue: UIStoryboardSegue, sender sender: AnyObject?) {
  switch UIStoryboard.Segue.Message(rawValue: segue.identifier)! {
  case .Custom:
    // Prepare for your custom segue transition
  case .NonCustom:
    // Pass in information to the destination View Controller
  }
}
```

_(which reminds me we should update the playgrounds one day…)_

### Example enum usage for Scenes

For Scenes it might makes sense for people to have a reference to a specific ViewController ref without instantiating right away

```switch
func navigate(to scene: StoryboardSceneType) {
  self.navigationController.push(scene.viewController)
}

private static let scenes = [StoryboardScene.Foo.foo, StoryboardScene.Foo.bar]

@IBAction func navigate() {
  go(to: scenes[picker.selectedIndex])
}
```